doc/source/tep/TEP14.md

```diff
@@ -225,3 +225,6 @@ Changes
 
 2016-03-16
 [cpascual](http://sf.net/u/cpascual/) Passing to  ACCEPTED
+
```

Comment:
A change entry should be added in **all** the TEPs indicating that URLs and format were updated according with TEP16